loky/process_executor.py

```diff
@@ -384,6 +390,7 @@ def _process_worker(call_queue, result_queue, initializer, initargs,
     _CURRENT_DEPTH = current_depth
     _process_reference_size = None
     _process_last_memory_check = None
+    _last_gc_call = None
```

Comment:
This global variable could also be merged with `process_last_memory_check`. Something like `_process_last_leak_check`